woocommerce/navigation
# Add site title component

Fixes #70 

Adds the site title component.

A follow will probably be needed:
* Site title/url should probably be migrated to `@wordpress/core-data`.

### Screenshots
<img width="297" alt="Screen Shot 2020-09-04 at 6 13 59 PM" src="https://user-images.githubusercontent.com/10561050/92288448-9b796f80-ef15-11ea-8abc-1e0d1a1bfd9e.png">
<img width="357" alt="Screen Shot 2020-09-04 at 6 13 52 PM" src="https://user-images.githubusercontent.com/10561050/92288449-9b796f80-ef15-11ea-915a-178a30be39a3.png">


### Testing
1. Open any page with the wc nav.
1. Make sure the nav is shown with your site title.
1. Click on the site title to make sure it navigates to your site's frontend.
1. Click on the site icon and make sure it folds the menu.

/cc @jameskoster This probably needs a design review.  The folded state in particular has a couple items that may need updating:
* The height of the header is 60px but the WCA header as seen above is 56px so they don't quite line up.
* Currently this folds with 60px remaining on the left (so there will always be a gap).  Is this okay or should it allow content to be flush left?  (We'll need to modify the WCA header slightly to accommodate if so.
